### DISCUSSION TITLE
Has anyone built a map mesh using truevision and successfully imported it into the Carla editor?

### DISCUSSION THREAD

**Discussion by Fitz-Lee** [carla-simulator/carla#7902] [Q&A]
The latest official documentation recommends using TrueVision instead of Roadrunner. However, TrueVision can only export .xodr files. But Carla needs .fbx and .xodr files. Is there a way to import maps into Carla using just .xodr files?

**Comment by PierreBEGOUT** [comment]
I am also wondering that, but I have import issue with TrueVision, my OpenDrive file seems to miss junctions. It is too bad that the standalone OpenDrive import in Carla doesn't provide base graphics.
